gtsam/linear/tests/testGaussianFactorGraph.cpp

```diff
@@ -18,16 +18,16 @@
  *  @author Richard Roberts
  **/
 
-#include <gtsam/linear/GaussianFactorGraph.h>
-#include <gtsam/linear/GaussianConditional.h>
-#include <gtsam/linear/GaussianBayesNet.h>
-#include <gtsam/inference/VariableSlots.h>
-#include <gtsam/inference/VariableIndex.h>
-#include <gtsam/base/debug.h>
-#include <gtsam/base/VerticalBlockMatrix.h>
-
-#include <gtsam/base/TestableAssertions.h>
 #include <CppUnitLite/TestHarness.h>
```

Comment:
The include order was already correct : please restore.

Reply:
Autoformat rearranged this. I'll restore it but wanted to let you know.

Reply:
Done